benchmark/fs/readfile.js

```diff
@@ -25,7 +25,7 @@ function main(conf) {
   bench.start();
   setTimeout(function() {
     bench.end(reads);
-    try { fs.unlinkSync(filename); } catch (e) {}
+    process.exit(0);
```

Comment:
ups, this was from some debugging. But the line after `bench.end(reads);` never executed before, because it called `process.exit(0)`.

I will update such it removes the file.